doc/rados/operations/pools.rst

```diff
@@ -320,103 +322,81 @@ You may set values for the following keys:
    :Type: Unsigned Integer
 
 .. _size:
 
 .. describe:: size
-
-   Sets the number of replicas for objects in the pool.
-   See `Set the Number of Object Replicas`_ for further details.
-   Replicated pools only.
-
+   
+   :Description: Sets the number of replicas for objects in the pool. For further details, see `Setting the Number of RADOS Object Replicas`_. Replicated pools only.
    :Type: Integer
 
 .. _min_size:
 
 .. describe:: min_size
-
-   Sets the minimum number of replicas required for I/O.
-   See `Set the Number of Object Replicas`_ for further details.
-   In the case of Erasure Coded pools this should be set to a value
-   greater than 'k' since if we allow IO at the value 'k' there is no
-   redundancy and data will be lost in the event of a permanent OSD
-   failure. For more information see `Erasure Code <../erasure-code>`_
-
+   
+   :Description: Sets the minimum number of replicas required for I/O.  For further details, see `Setting the Number of RADOS Object Replicas`_.  For erasure-coded pools, this should be set to a value greater than 'k'. If I/O is allowed at the value 'k', then there is no redundancy and data will be lost in the event of a permanent OSD failure. For more information, see `Erasure Code <../erasure-code>`_
    :Type: Integer
    :Version: ``0.54`` and above
 
 .. _pg_num:
 
 .. describe:: pg_num
-
-   The effective number of placement groups to use when calculating
-   data placement.
-
+   
+   :Description: Sets the effective number of PGs to use when calculating data placement.
    :Type: Integer
-   :Valid Range: Superior to ``pg_num`` current value.
+   :Valid Range: ``0`` to ``mon_max_pool_pg_num``. If set to ``0``, the value of ``osd_pool_default_pg_num`` will be used. 
 
 .. _pgp_num:
 
 .. describe:: pgp_num
-
-   The effective number of placement groups for placement to use
-   when calculating data placement.
-
+   
+   :Description: Sets the effective number of PGs to use when calculating data placement.
    :Type: Integer
-   :Valid Range: Equal to or less than ``pg_num``.
+   :Valid Range: Between ``1`` and the current value of ``pg_num``.
 
 .. _crush_rule:
 
 .. describe:: crush_rule
-
-   The rule to use for mapping object placement in the cluster.
-
+   
+   :Description: Sets the CRUSH rule that Ceph uses to map object placement within the pool.
    :Type: String
 
 .. _allow_ec_overwrites:
 
 .. describe:: allow_ec_overwrites
-
-
-   Whether writes to an erasure coded pool can update part
-   of an object, so cephfs and rbd can use it. See
-   `Erasure Coding with Overwrites`_ for more details.
-
+   
+   :Description: Determines whether writes to an erasure-coded pool are allowed to update only part of a RADOS object. This allows CephFS and RBD to use the pool for data (as opposed to metadata). For more details, see `Erasure Coding with Overwrites`_.
```

Comment:
My "as opposed to" comment was worded for discussion.  For inclusion here, I suggest

>This allows CephFS and RBD to use an EC pool for user data (but not for metadata).